swathi-practice/check_upper_lower.java

```diff
@@ -0,0 +1,22 @@
+import java.util.*;
+public class check_upper_lower
```

Comment:
Rename with Check_upper_lower. Class name should be starting with a capital letter

Reply:
this comment is valid throughout the commit